docs/kcl-lang/functions.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Functions"
+excerpt: "Documentation of the KCL language for the Zoo Design Studio."
+layout: manual
+---
+
+We have support for defining your own functions. Functions can take in any
+type of argument. Below is an example of the syntax:
+
+```
+fn myFn(x) {
+  return x
+}
+```
+
+As you can see above `myFn` just returns whatever it is given.
+
+KCL's early drafts used positional arguments, but we now use keyword arguments:
```

Comment:
I'd remove this line, we're all-in on keyword arguments now. Let the past die. Kill it, if you have to.